### DISCUSSION TITLE
Problem of not being able to have multiple rounds of conversation using the openai interface

### DISCUSSION THREAD

**Discussion by muzhi1991** [open-webui/open-webui#947] [General]
When I use openai’s interface for multi-round conversations, I can’t initiate a request after a few rounds of conversations, and I get the following error：

chrome console log：
```
5.31442014.js:5 Uncaught (in promise) ReferenceError: OPENAI_API_BASE_URL is not defined
    at Y (5.31442014.js:5:2692)
    at 5.31442014.js:2:482
    at Array.map (<anonymous>)
    at ce (5.31442014.js:2:158)
    at Ee (5.31442014.js:1:7016)
```
why OPENAI_API_BASE_URL var not exists, The problem seems to occur with the following code：

https://github.com/open-webui/open-webui/blob/6df2505bf0352a7580b33f17ce6844afe04fb7be/src/routes/(app)/%2Bpage.svelte#L588

I don't know much about js, how do I fix it?

**Comment by justinh-rahb** [comment]
How many turns are we talking about? Are you using OpenAI API through LiteLLM?

**Comment by muzhi1991** [comment]
> How many turns are we talking about? Are you using OpenAI API through LiteLLM?
about 3 turns（3 request  2 response，block in third turns）
no use LiteLLM, I only set api-key in Connecttion Settings, choose gpt-3.5-turbo

**Comment by muzhi1991** [comment]
<img width="1011" alt="image" src="https://github.com/open-webui/open-webui/assets/2101044/79fa422c-0fdc-4304-857c-c86a132759a5">
I can't find this variable in the file search。

To add, I'm using the docker version, is it the js compilation that replaces the variables？

**Comment by tjbck** [comment]
Unable to reproduce the issue, moving to discussion.

**Comment by yfzhou0904** [comment]
I'm observing a similar issue. It seems to occurs when the page is refreshed (or the connection is dropped inadvertently) and I try to continue the conversation afterwards.

To reproduce: start a new chat with an OpenAI gpt model -> get some response -> refresh the page to the same chat -> send another message -> no response, this is printed in console:
<img width="497" alt="Screenshot 2024-02-28 at 17 48 04" src="https://github.com/open-webui/open-webui/assets/13145408/555dab10-6212-41e0-b85b-4044c98417db">

**Reply by tjbck** [reply]
That helped me to isolate the problem! #955 should fix it, Thanks!

**Reply by muzhi1991** [reply]
wonderful
